### PR TITLE
Add example response in webhook documentation

### PR DESCRIPTION
**Pull request by markovska**
Add an example code block for fetching webhook headers after making API payment request.

### CODE REVIEW

File: source/guides/webhooks.rst [hunk 1]
```diff
@@ -13,6 +13,18 @@ will call that webhook URL with a single POST-parameter called ``id`` and a valu
 script behind your webhook URL should use that ID to :doc:`fetch the payment status </reference/v2/payments-api/get-payment>`
 and act accordingly. If the new payment status is ``paid`` you can start shipping the order.
 
+.. code-block:: bash
```
Comment (shirshir) [inline source/guides/webhooks.rst]: I would align all lines in the request example to left, and remove the leading spaces.

File: source/guides/webhooks.rst [hunk 2]
```diff
@@ -13,6 +13,18 @@ will call that webhook URL with a single POST-parameter called ``id`` and a valu
...
+      :linenos:
+
+          POST / HTTP/1.0
+            Host: https://webshop.example.org/payments/webhook
```
Comment (shirshir) [inline source/guides/webhooks.rst]: I think the `Host:` line will only contain the host (`webshop.example.org`), and the line with `POST` will contain the path: `POST /payments/webhook HTTP/1.0`

File: source/guides/webhooks.rst [hunk 3]
```diff
@@ -13,6 +13,18 @@ will call that webhook URL with a single POST-parameter called ``id`` and a valu
...
+            Via: 1.1 tinyproxy (tinyproxy/1.8.3)
+            Content-Type: application/x-www-form-urlencoded
+            Accept: */*
+            Accept-Encoding: deflate, gzip
+            Content-Length: 16
+            id=tr_d0b0E3EA3v
```
Comment (shirshir) [inline source/guides/webhooks.rst]: Maybe add a newline so it is clear that the `id=tr_d0b0E3EA3v` line is the request body.

File: source/guides/webhooks.rst (hunk 1 above)
Comment (markovska) [reply source/guides/webhooks.rst]: Updated

File: source/guides/webhooks.rst (hunk 2 above)
Comment (markovska) [reply source/guides/webhooks.rst]: Updated

File: source/guides/webhooks.rst [hunk 4]
```diff
@@ -13,6 +13,19 @@ will call that webhook URL with a single POST-parameter called ``id`` and a valu
 script behind your webhook URL should use that ID to :doc:`fetch the payment status </reference/v2/payments-api/get-payment>`
 and act accordingly. If the new payment status is ``paid`` you can start shipping the order.
 
+.. code-block:: bash
+      :linenos:
+
+      POST /payments/webhook HTTP/1.0
+      Host: https://webshop.example.org
```
[the remark this answers is not in the document]
Comment (markovska) [reply source/guides/webhooks.rst]: missed that, should be ok now.

**Comment by shirshir** [comment]
Thanks @markovska !

**Pull request merged**